WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -77,7 +77,7 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        title = NSLocalizedString("Pages", comment: "Tile of the screen showing the list of pages for a blog.")
+        title = NSLocalizedString("Site Pages", comment: "Tile of the screen showing the list of pages for a blog.")
```

Comment:
Sorry, going to nitpick – I just noticed the comment here says `Tile` not `Title` 😬